app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+
+    if user&.authenticate params[:session][:password]
```

Comment:
toán tử `&.` có tác dụng gì em?

Reply:
chưa trả lời câu hỏi nè e

Reply:
trả về true false khi thực thi hàm authenticate password đã mã hóa ạ

Reply:
trả lời sai trọng tâm rùi nhé, chị đang hỏi tác dụng của toán tử `&.`
nếu ko dùng toán tử này thì có cách nào khác để check điều kiện chỗ này ko em?

Reply:
để đảm bảo tìm thấy đối tượng user
sau đó, sẽ kiểm tra xem có trùng password hay không, nếu trùng sẽ cho phép login ạ